.github/workflows/create-releases.yml

```
@@ -43,7 +43,7 @@ jobs:
     
     # Build
     - name: Set up JDK 1.8
-      uses: actions/setup-java@v2.1.0
+      uses: actions/setup-java@v2.2.0
```

You can directly use the major version
```suggestion
      uses: actions/setup-java@v2
```

---

I think this is a good idea, as there is already a v2.3.0 release of setup-java@v2.